docs/source/_static/bibliography.bib

```diff
@@ -125,38 +120,81 @@ @article{Markall2010
+  title =        {Performance-Portable Finite Element Assembly Using
+                  PyOP2 and FEniCS},
+  booktitle =    {28th International Supercomputing Conference, ISC,
+                  Proceedings},
+  pages =        {279-289},
+  year =         2013,
+  editor =       {Kunkel, Julian Martin and Ludwig, Thomas and Meuer,
+                  Hans Werner},
+  volume =       7905,
+  series =       {Lecture Notes in Computer Science},
+  publisher =    {Springer},
+  doi =          {10.1007/978-3-642-38750-0_21},
   url =          "http://dx.doi.org/10.1007/978-3-642-38750-0_21"
 }
 
+@article{McRae2015,
+  title =        {Automated generation and symbolic manipulation of
+                  tensor product finite elements},
+  author =       {McRae, Andrew T. T. and Bercea, Gheorghe-Teodor and
+                  Mitchell, Lawrence and Ham, David A. and Cotter,
+                  Colin J.},
+  archivePrefix ="arXiv",
+  journal =      {Submitted to SIAM Journal on Scientific Computing},
+  eprint =       {1411.2940},
+  year =         2015,
+  url =          {http://arxiv.org/abs/1411.2940}
+}
+
 @InProceedings{Rathgeber2012,
-author = {Florian Rathgeber and Graham R. Markall and Lawrence Mitchell and Nicolas Loriant and David A. Ham and Carlo Bertolli and Paul H.J. Kelly},
-title = {PyOP2: A High-Level Framework for Performance-Portable Simulations on Unstructured Meshes},
-booktitle ={High Performance Computing, Networking Storage and Analysis, SC Companion:},
-isbn = {978-1-4673-3049-7},
-year = {2012},
-pages = {1116-1123},
-doi = {10.1109/SC.Companion.2012.134},
-publisher = {IEEE Computer Society},
-address = {Los Alamitos, CA, USA},
+  author =       {Florian Rathgeber and Graham R. Markall and Lawrence
+                  Mitchell and Nicolas Loriant and David A. Ham and
+                  Carlo Bertolli and Paul H.J. Kelly},
+  title =        {PyOP2: A High-Level Framework for
+                  Performance-Portable Simulations on Unstructured
+                  Meshes},
+  booktitle =    {High Performance Computing, Networking Storage and
+                  Analysis, SC Companion:},
+  isbn =         {978-1-4673-3049-7},
+  year =         2012,
+  pages =        {1116-1123},
+  doi =          {10.1109/SC.Companion.2012.134},
+  publisher =    {IEEE Computer Society},
+  address =      {Los Alamitos, CA, USA},
   url =          "http://dx.doi.org/10.1109/SC.Companion.2012.134"
 }
+
+@article{Rathgeber2015,
+  title =        {Firedrake: automating the finite element method by
+                  composing abstractions},
+  author =       {Rathgeber, Florian and Ham, David A. and Mitchell,
+                  Lawrence and Lange, Michael and Luporini, Fabio and
+                  McRae, Andrew T. T. and Bercea, Gheorghe-Teodor and
+                  Markall, Graham R. and Kelly, Paul H. J.},
+  archivePrefix ="arXiv",
+  journal =      {Submitted to ACM TOMS},
+  eprint =       {1501.01809},
+  year =         2015,
```

Comment:
I never know what year to give preprints.  The preprint (even v2 on the arxiv) is 2015, I guess when it appears it will probably be 2016.